pretty.js

```diff
@@ -155,17 +177,43 @@ function pretty (opts) {
     return line
   }
 
-  function asISODate (time) {
-    return new Date(time).toISOString()
+  function asISODate (aTime, aFmt, aTZO) {
+    var time = aTime
+    var format = aFmt || 'YYYY-MM-DDThh:mm:ss.SSSTZ'
+
+    var date = new Date(time)
+    // make independent of the system timezone
+    var tzOffset = aTZO || date.getTimezoneOffset()
+    date.setUTCMinutes(date.getUTCMinutes() - tzOffset)
+    var year = format.indexOf('YYYY') > -1
+      ? date.getUTCFullYear()
+      : date.getUTCFullYear().toString().substring(2, 4)
+    var month = _lpadzero(date.getUTCMonth() + 1, 2)
+    var day = _lpadzero(date.getUTCDate(), 2)
+    var hour = _lpadzero(date.getUTCHours(), 2)
+    var minute = _lpadzero(date.getUTCMinutes(), 2)
+    var second = _lpadzero(date.getUTCSeconds(), 2)
+    var milli = _lpadzero(date.getUTCMilliseconds(), 3)
+    date.setUTCMinutes(date.getUTCMinutes() + tzOffset)
+
+    var _format = format
+      .replace(/Y{1,4}/g, year)
+      .replace(/MM/g, month)
+      .replace(/DD/g, day)
+      .replace(/hh/g, hour)
+      .replace(/mm/g, minute)
+      .replace(/ss/g, second)
+      .replace(/SSS/g, milli)
+      .replace(/TZ/g, toTimezoneOffset(tzOffset))
+    return _format
```

Comment:
are we sure there is no common utility we can use instead of this block of replace?

Reply:
I think this is minimal implemantation without any deps.
this is a log libary not a datetime libary !? XD
there must have another fast convert implemantation

Reply:
Can you make the _previous code_ as the default (`new Date(time).toISOString()`)?

Reply:
sure, add new function asLocalISODate and keep asISODate as the previouse code
and add CLI options (`--localTime`) transform to localISOTime with tz(`YYYY-MM-DDThh:mm:ss.SSSTZ`)
but, not sure how to add unit test for `asLocalISODate`, only bin.js for pretty print